views/partials/email_partials/email_single_breach.hbs

```diff
@@ -0,0 +1,13 @@
+    <tr class="email-breach-listing">
+      <td class="email-breach-image" valign="top" style="padding-top: 13px; padding-bottom: 10px; width: 55px;">
+        <img src="{{ SERVER_URL }}/img/logos/{{ this.Name }}.png" width="40" style="width: 40px" />
```

Comment:
I can't remember from the other PRs, but does this take into account a new breach named "Foo" but we don't have any logos for it yet? Or will this send out a link to "Foo.png" which will 404 in the email?

Reply:
In a report scenario, it will 404 all over the place. The breach alerts are hard-coded to show the missing-image icon for now.